integrations/document-threshold.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: integration
+name: Document Threshold
+description: This component filters documents based on a threshold percentage, ensuring only the documents above the threshold get passed down the pipeline.
```

Comment:
Is it a confidence score percentage? If yes let's say "Filter documents based on a confidence score threshold"

Reply:
Changed.